run.py

```diff
@@ -41,10 +44,17 @@ def parse_arguments():
     # Call function to create Kubernetes view
     logging.info(f"Creating Kubernetes view for {cluster}")
     file_name = create_k8s_view(cluster, label_name, label_value, timestamp)
+    if file_name is None:
+        logging.info("No node found to drain. Exiting...")
+        sys.exit(0)
     logging.info(f"Kubernetes view created successfully and written to {file_name}")
     logging.info(f"Creating action plan for {cluster}")
     node_name, action_plan_path = create_action_plan(file_name, timestamp)
+    status = False
     if node_name is None:
         logging.info("No node found to drain. Exiting...")
     else:
-        drain_node(cluster, node_name, action_plan_path, 'False', dry_run)
+        status = drain_node(cluster, node_name, action_plan_path, 'False', dry_run)
+    if status and dry_run == 'False':
+        time.sleep(60)  # Sleep for 60 seconds
+        subprocess.run(["python3", "run.py", "-c", cluster, "-l", label_name, "-v", label_value, "--log", log_level, "--dry_run", dry_run])
```

Comment:
@roya2yush Better to use a while loop and a max retries counter so that this doesn't run forever forking a new process every 60 seconds and consuming all the RAM or running out of max processes (32,768 in about 23 days).